spec/fixtures/team_with_one_application.json

```
@@ -4,6 +4,7 @@
   "team":"#asset-management",
   "dependencies_team":"#govuk-pub-workflow",
   "puppet_name":"asset_manager",
+  "cd_enabled": true,
```

```suggestion
  "continuously_deployed": true,
```

I have a suggestion, just to avoid an acronym that not everyone might be familiar with.